akka-testkit-typed/src/main/scala/akka/testkit/typed/scaladsl/TestProbe.scala

```diff
@@ -241,6 +248,24 @@ class TestProbe[M](name: String)(implicit system: ActorSystem[_]) {
     o.asInstanceOf[C]
   }
 
+  /**
+   * Expect the given actor to be stopped or stop withing the given timeout or
+   * throw an AssertionFailure.
```

Comment:
it's an `AssertionError`, not Failure :)

Reply:
Failure to use the right name!